bibtex-actions.el

```diff
@@ -324,10 +324,10 @@ is non-nil, also run the `bibtex-actions-before-refresh-hook' hook."
   ;; itself is used to indicate a cache that hasn't been computed this is to
   ;; work around the fact that we can't differentiate between empty list and nil
```

Comment:
This comment is outdated now. I forgot to remove it earlier.

Reply:
Oops; I merged it.

You can remove it when you do the related PR :-)

Reply:
That was quick. I hope these late changes don't introduce any errors. They are pretty trivial but still fingers crossed.